Makefile

```diff
@@ -0,0 +1,335 @@
+.PHONY: conda-app conda-environment ta-lib conda-environment-update docker-app \
+	build-docker-app docker-start docker-freqtrade docker-stop docker-clean \
+	docker-state docker-inspect docker-logs pyclean pylint
+################################################################################
+#  Project Parameters and commands                                             #
+################################################################################
+# TODO https://gist.github.com/DerekV/3030284
+############################# ###############################
+# Specify shell
+SHELL := /bin/bash
+
+# Define the project root dir
+PROJECT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+# Detect operating system in Makefile.
+OSFLAG :=
+	UNAME_S := $(shell uname -s)
+	ifeq ($(UNAME_S),Linux)
+		OSFLAG += LINUX
+	endif
+	ifeq ($(UNAME_S),Darwin)
+		OSFLAG += OSX
+	endif
+	ifeq ($(UNAME_S),Windows*)
```

Comment:
I am not certain that this makefile works for windows...

many commands won't work (`find` for example), but also the `make ta-lib` won't work.
i'm not that familiar with makefiles to determine if they should check that for each command or not